impeller/aiks/canvas.cc

```diff
@@ -363,8 +363,15 @@ void Canvas::SaveLayer(
   Save(true, paint.blend_mode, backdrop_filter);
 
   auto& new_layer_pass = GetCurrentPass();
-  new_layer_pass.SetDelegate(
-      std::make_unique<PaintPassDelegate>(paint, bounds));
+
+  // Only apply opacity peephole on default blending.
```

Comment:
Does this deserve a TODO/bug to extend this to other blend modes?

Reply:
I don't think we can in general.